eslint.config.js

```diff
@@ -3,21 +3,30 @@ import globals from 'globals'
 import reactHooks from 'eslint-plugin-react-hooks'
 import reactRefresh from 'eslint-plugin-react-refresh'
 import tseslint from 'typescript-eslint'
+import prettier from 'eslint-config-prettier'
+import prettierPlugin from 'eslint-plugin-prettier'
 import { defineConfig, globalIgnores } from 'eslint/config'
 
 export default defineConfig([
-  globalIgnores(['dist']),
+  globalIgnores(['dist', 'node_modules', 'build', 'dist-ssr', '*.local']),
   {
     files: ['**/*.{ts,tsx}'],
     extends: [
       js.configs.recommended,
-      tseslint.configs.recommended,
+      ...tseslint.configs.recommended,
       reactHooks.configs.flat.recommended,
       reactRefresh.configs.vite,
+      prettier,
     ],
+    plugins: {
+      prettier: prettierPlugin,
+    },
     languageOptions: {
       ecmaVersion: 2020,
       globals: globals.browser,
     },
+    rules: {
+      'prettier/prettier': 'error',
+    },
```

Comment:
```suggestion
    rules: {
      'prettier/prettier': 'error',
      // 2번: curly - 모든 제어문에 중괄호 필수
      curly: ['error', 'all'],
      // 3번: no-console - console 사용 시 경고
      'no-console': 'warn',
      // 4번: TypeScript 모듈 경계 타입 명시 비활성화
      '@typescript-eslint/explicit-module-boundary-types': 'off',
      // 5번: 미사용 변수 경고 (_, args로 시작하는 인자 제외)
      '@typescript-eslint/no-unused-vars': [
        'warn',
        { argsIgnorePattern: '^_|^args$' },
      ],
      // 6번: React 17+ JSX 변환 지원
      'react/react-in-jsx-scope': 'off',
      // 8번: import 정렬 규칙 (프로젝트 구조에 맞게 수정)
      'simple-import-sort/imports': [
        'error',
        {
          groups: [
            ['^node:'],
            ['^react$', '^@?\\w'],
            ['^@app/'],
            ['^@assets/'],            
            ['^@pages/'],
            ['^@widgets/'],
            ['^@shared/'],
            ['^\\u0000'],
            ['^\\.'],
            ['^.+\\.css$'],
          ],
        },
      ],
      // 9번: export 정렬 규칙
      // 
      'simple-import-sort/exports': 'error',
    },
```
위 내용들 추가하면 어떨까요??

만약 추가하면 8,9번은 `simple-import-sort` 플러그인 설치해야 돼요!

Reply:
추가하면 주석은 번호만 지우고 남겨둬도 괜찮을것 같아요!